packages/application-components/src/components/dialogs/info-dialog/README.md

```diff
@@ -0,0 +1,41 @@
+# InfoDialog
+
+## Usage
+
+```js
+import { InfoDialog } from '@commercetools-frontend/application-components';
+```
+
+#### Description
+
+Info dialogs are controlled components used to show more information about a particular feature, UI element, etc. They are mainly composed of text paragraphs.
+
+#### Usage
+
+```js
+<InfoDialog
+  title={<FormattedMessage {...messages.infoTitle} />}
+  isOpen={isOpen}
+  onClose={handleClose}
+>
+  <Spacings.Stack scale="m">
+    <div>
+      <Text.Body>{'Lorem ipsus ...'}</Text.Body>
+    </div>
+    <div>
+      <Text.Body>{'Lorem ipsus ...'}</Text.Body>
+    </div>
+  </Spacings.Stack>
+</InfoDialog>
+```
+
+#### Properties
+
+| Props                  | Type       | Required | Values   | Default | Description                                                                                                                 |
+| ---------------------- | ---------- | :------: | -------- | ------- | --------------------------------------------------------------------------------------------------------------------------- |
+| `title`                | `node`     |    ✅    | -        | -       | The title of the Info Dialog                                                                                                |
+| `isOpen`               | `boolean`  |    ✅    | -        | -       | Indicates whether the dialog should open or not                                                                             |
```

Comment:
I'm not really sure what this prop means. Does passing `isOpen` mean that it has to be controlled by the prop? Or does passing `{isOpen=true}` set a kind of "initial open state", where afterwards the opening and closing is managed by the component and not the parent?

Reply:
It's a controlled component, I mentioned that in the README description. How can we make it more clear?

Reply:
`Indicates the initial open state of the dialog`?

Reply:
Hmm wait, the `isOpen` state needs to be managed outside of the dialog component. There is no "initial" state.

Reply:
`Indicates whether the dialog is open or closed` then?